dist/gaia-header.js

```diff
@@ -913,11 +918,40 @@ module.exports = component.register('gaia-header', {
   observerStop: function() {
     if (!this.observing) { return; }
     this.observer.disconnect();
+    window.removeEventListener('resize', this);
+    this._resizeHandled = false;
+
     this.observing = false;
     debug('observer stopped');
   },
 
   /**
+   * Handle DOM events
+   */
+  handleEvent: function(e) {
```

Comment:
me neither but if we want to remove the event listener while still retaining the this, we need either a pre-bound function, or use this pattern. I thought that in this case it was easier to do it like this.